_sass/includes/header-navigation.scss

```diff
@@ -1,8 +1,22 @@
+body.guides,
```

Comment:
In the list of guides http://localhost:4000/guides/
Look for the bottom of the page,

> When measuring the footprint of a Quarkus application, we measure Resident Set Size (RSS) and not the JVM heap size which is only a small part of the overall problem. The JVM not only allocates native memory for heap (-Xms, -Xmx) but also structures required by the jvm to run your application.

The element in parenthesis is code (marked class highlighter-rouge) and does not render well in the black background. I think we might need different asciidoc css depending on the light or dark background